quiche/src/recovery/gcongestion/bbr2.rs

```diff
@@ -422,7 +422,7 @@ impl BBRv2 {
             mode: Mode::startup(BBRv2NetworkModel::new(&params, smoothed_rtt)),
             cwnd,
             pacing_rate: Bandwidth::from_bytes_and_time_delta(cwnd, smoothed_rtt) *
-                2.885,
+                params.startup_pacing_gain,
```

Comment:
This is changing the default value but quiche is used by other teams so are we sure its a safe change? Where did the previous value come from?

---
Changes to default values should be in a separate commit.

Can you add a comment [here](https://github.com/cloudflare/quiche/blob/37defda36f23a351eacd36667eda0b6edde72d44/quiche/src/recovery/gcongestion/bbr2.rs#L232C26-L232C32) specify where that value is coming from (link to kernel source or bbr spec)

Reply:
There are 4 commits in this PR in an attempt to separate semantic fixes, from spec consistency and new parameters.  I split the last commit into a separate PR since it is the change I care the most about at this time and hopefully not a controversial change.  See https://github.com/cloudflare/quiche/pull/2115